packages/monorepo-tools/README.md

```diff
@@ -123,6 +123,18 @@ Delete all local branches and create all non-remote-tracking branches of a speci
 
```

Comment:
We have new function :) New package is allowed by split of monorepo. I would like to see a brief tutorial on how to split the new package. I need only two or three sentences in the new section before the Reference section